app/models/audition.rb

```diff
@@ -1,3 +1,14 @@
 class Audition < ActiveRecord::Base
-  
-end
\ No newline at end of file
+  belongs_to :role
+
+  def audition_role
+    Role.find(role_id)
+  end
```

Comment:
We def dont need this anymore, these fall into the instance methods that i mentioned we get out of the box. the example i posted has the little note on it but we should not be needing these it was only a written for us to check to see that it works.